managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -1562,6 +1562,20 @@ private void updateLedgersListAfterRollover(MetaStoreCallback<Void> callback) {
         store.asyncUpdateLedgerIds(name, getManagedLedgerInfo(), ledgersStat, callback);
     }
 
+    private void updateLedgersListAfterRollover(MetaStoreCallback<Void> callback, ManagedLedgerInfo mlInfo) {
```

Comment:
It seems we can just modify in previous method `updateLedgersListAfterRollover`, not creating new one.

Reply:
fixed.